index.html

```diff
@@ -367,8 +367,10 @@ <h4 data-i18n="index.page.requirements.updates.header">Frequent Updates</h4>
 </script>
 
 <script>
-    var ext = "";
-    var OS = "UNKNOWN";
+    // Detect Operating System and required file extension
+    var ext = "",
+        OS = "UNKNOWN";
+
     if (/Windows|Win32|WOW64|Win64/.test(navigator.userAgent)) {
```

Comment:
If you wanted to be more generic and possibly catch any changes (if there are ever changes), you could use

`if (w[in|ow]/i.test(navigator.userAgent)) {`

In fact, it'd probably be a good idea to make these checks case-insensitive.